tests/validation/suite_test.go

```diff
@@ -12,6 +12,7 @@ import (
 
 func TestMain(m *testing.M) {
 	testutil.ValidationPreCheck()
+	testutil.KubectlCmd([]string{"create", "namespace", testutil.TestingNamespace})
```

Comment:
Whats this for ? If it's for previous flakey tests that should be resolved already.

Reply:
Yes this is to resolve some of the flakiness. Does #701 resolve? I noticed that tests sometimes fail in a new cluster (or cluster that just doesn't have the `testutil.TestingNamespace` namespace) due to some sort of race in the parallel tests all trying to create it at once.

Reply:
It was an earlier PR that fixed it, should be good to remove this. If it acts up in future we can re-add.